source/release-notes.txt

```diff
@@ -10,6 +10,27 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.42.1
+----------------
+
+*Released February 15, 2024*
+
+New Features
+
+- Added vector search index creation to the create search indexes 
+  modal :issue:`COMPASS-7302`.
```

Comment:
```suggestion
  modal (:issue:`COMPASS-7302`).
```

Reply:
[suggestion] adding parenthesis around the JIRAs to remain consistent with the other release notes on the page

Reply:
Agree, good catch thanks @ltran-mdb2